isolationsegments.go

```diff
@@ -100,10 +100,13 @@ func (c *Client) GetIsolationSegmentByGUID(guid string) (*IsolationSegment, erro
 
 func (c *Client) ListIsolationSegmentsByQuery(query url.Values) ([]IsolationSegment, error) {
 	var iss []IsolationSegment
-	requestUrl := "/v3/isolation_segments?" + query.Encode()
+	requestURL := "/v3/isolation_segments" + query.Encode()
```

Comment:
You're appending the query string twice here.

Reply:
good catch, thanks!